rails-commenting.rb

```diff
@@ -6,25 +6,26 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) Created a class called BlogPostsController that is inheriting from ApplicationController. 
 class BlogPostsController < ApplicationController
+  
   def index
-    # ---2)
+    # ---2) Created an instance variable called @posts and assigned it to call on all items in the database (Blogpost.all). By making posts an instance variable we can now call it outside of the index method. 
     @posts = BlogPost.all
   end
 
-  # ---3)
+  # ---3) Created a method called show that tells Rails to find only the post that has the id defined in the params. 
   def show
     @post = BlogPost.find(params[:id])
   end
 
-  # ---4)
+  # ---4) Created a method called new, that creates a new post inside the Blogpost when called. 
```

Comment:
This one is often tricky as this is not actually updating the database, this method will correspond to the form.